src/main/java/frc/robot/subsystems/SwerveSubsystem.java

```diff
@@ -171,7 +199,9 @@ else if(rotationHold != null){
         }
         SmartDashboard.putString("chassis speeds",chassisSpeeds.toString());
         //Convert Chassis Speeds to individual module states
-        return kDriveKinematics.toSwerveModuleStates(chassisSpeeds);
+        SwerveModuleState[] moduleStates = kDriveKinematics.toSwerveModuleStates(chassisSpeeds);
+        Logger.getInstance().recordOutput("Desired States", moduleStates);
```

Comment:
Why are we logging like this?  Is this a string?  Is this for advantagescope?  

Would prefer numbers instead of strings.

Reply:
and we already log desiredoutput in swervemodule

Reply:
( Talked to zane, this is advantagescope required. )